Module for SAP "RECON" CVE-2020-6287 that adds an admin user (#13852)
rapid7/metasploit-framework: documentation/modules/auxiliary/admin/sap/cve_2020_6287_ws_add_user.md

> ## Vulnerable Application

Reviewer: ```suggestion
## Vulnerable Application

```

Minor but consistent.

This module leverages an unauthenticated web service to submit a job which will create a user with a specified role. The
job involves running a wizard. After the necessary action is taken, the job is canceled to avoid unnecessary system
changes.

SAP NetWeaver NetWeaver versions 7.30 through 7.50 are affected by this vulnerability. An Amazon Machine Image (AMI) for
Amazon Web Services (AWS) can be used as a testing environment. One such image is provided by Linke IT America LLC and
is available on the [AWS Marketplace][1] with installation instructions posted to their [blog][2].

> Once setup and configured, the instances will be vulnerable on the default HTTP port 50000.

Reviewer: ```suggestion
Once set up and configured, the instances will be vulnerable on the default HTTP port 50000.
```

Verb form.

If the password does not meet the requirements (e.g. the value is too short), the server will respond with an error
message and the Metasploit module will need to be rerun.

## Verification Steps

  1. Install the application
  1. Start msfconsole
  1. Do: `use auxiliary/admin/sap/cve_2020_6287_ws_add_user`
  1. Set the `RHOST`, `USERNAME`, and `PASSWORD` options
  1. Run the module and wait a few seconds
  1. Once the "PCK Upgrade" job has been canceled, log in with the created credentials

## Options

> ### ROLE

Reviewer: ```suggestion
### ROLE

```
